src/cache/precomputed_quantities.jl

```diff
@@ -502,6 +512,19 @@ NVTX.@annotate function set_explicit_precomputed_quantities!(Y, p, t)
 
     if turbconv_model isa PrognosticEDMFX
         set_prognostic_edmf_precomputed_quantities_bottom_bc!(Y, p, t)
+    elseif turbconv_model isa DiagnosticEDMFX
+        set_diagnostic_edmf_precomputed_quantities_bottom_bc!(Y, p, t)
```

Comment:
The `bottom_bc` should be after `draft` (otherwise it is overwritten by the `draft` function). I think the easiest way to make this work is to move `bottom_bc` to implicit tendency, but after `draft`.

Reply:
resolved offline: this works too.